pkgs/development/interpreters/proglodyte-wasm/default.nix

```diff
@@ -30,7 +30,8 @@ let
 in
 
 stdenv.mkDerivation {
-  name = "wasm-0.0.1";
```

Comment:
We might as well drop this. The repo has 0 stars and is unmaintained since 5 years.